locksmith/src/controllers/hookController.ts

```diff
@@ -53,15 +53,24 @@ export async function subscribe(
     return createdHook
   }
 
-  // If changing mode, reconfirm the intent and update
-  if (hook?.mode !== hub.mode) {
+  // If changing mode or lease, reconfirm the intent and update
+  if (hook?.mode !== hub.mode || hub.lease_seconds) {
```

Comment:
They can also update the secret... (important to support that for security reasons...)

Reply:
okay will add that.